README.md

```diff
@@ -20,6 +20,11 @@ weggli is inspired by great tools like [Semgrep](https://semgrep.dev/), [Coccine
 - **Greedy**: weggli's pattern matching is designed to find as many (useful) matches as possible for a specific query. While this increases the risk of false positives it simplifies query creation. For example, the query  `$x = 10;` will match both assignment expressions (`foo = 10;`) and declarations (`int bar = 10;`). 
 
 
+## Install
```

Comment:
Nit: Can you move this down in front of the build instructions? :)

Reply:
done